packages/sampler/stories/qa/IncrementSlider.js

```diff
@@ -111,4 +138,13 @@ storiesOf('IncrementSlider', module)
 				<IncrementSliderWithContextualPopup />
 			</div>
 		)
+	)
+	.add(
+		'with max, min, and value',
+		() => (
+			<div>
+				Test the IncrementSlider by changing the values of min and value knobs.
```

Comment:
`min and value knobs.` -> `max, min, and value knobs.`. Please update the description for the sampler.